app/storage/encrypted_storage.py

```diff
@@ -1,34 +1,56 @@
-import json
-
 from app import settings
 from app.cryptography.jwe_decryption import JWEDirDecrypter
 from app.cryptography.jwe_encryption import JWEDirEncrypter
-from app.storage.abstract_server_storage import AbstractServerStorage, logger
+from app.storage.abstract_server_storage import AbstractServerStorage
+from app.utilities.strings import to_str
+from app.utilities.strings import to_bytes
+import json
+import hashlib
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 class EncryptedServerStorageDecorator(AbstractServerStorage):
 
     ENCRYPTED_KEY = 'encrypted'
```

Comment:
Really need some comments around this value, what is it for, on face value looks like a security problem but that's probably because it isn't clear wha it's for.